Dockerfile

```diff
@@ -1,60 +1,44 @@
-# This image provides a Python 3.6 environment you can use to run your Python
-# applications.
-FROM centos/s2i-base-centos7
+FROM centos/python-36-centos7
 
 EXPOSE 8080
 
-ENV PYTHON_VERSION=3.6 \
-    NODEJS_VERSION=8 \
+ENV NODEJS_VERSION=8 \
+    NODEJS_SCL=rh-nodejs8 \
     NPM_RUN=start \
     NAME=nodejs \
```

Comment:
Wonder if this `NAME` value I brought over from the NODEJS container is unnecessary or could be _koku_?